### DISCUSSION TITLE
Интересные GIFки для конвертации через Jinx в эффекты

### DISCUSSION THREAD

**Discussion by saippuakauppias** [vvip-68/LedPanelWiFi#50] [Похвастушки]
Давайте тут делиться наборами гифок, которые можно сконвертировать через Jinx! в эффект, загружаемый в SD-карту :) 

Я нашел кучку гифок + наконвертил из видео с ютуба (скачивал через какой-то сайт, конвертил mp4 через https://ezgif.com/video-to-gif , там же кропал и менял FPS/длинну гифки ).

Хочется ещё каких-нибудь эффектов, но уже устал искать (особенно хочется "мемов"). Авось кто-то ещё поделится :) 

Мой пак гифок будет лежать здесь: https://disk.yandex.ru/d/1DYvkYP_kj3RTQ

PS: делиться готовыми `.out` файлами не вижу смысла, размеры матрицы у всех разные. Сконвертить не сложно, буквально пару кнопок (Image view или как-то так + выбрать путь к gif) :)

**Comment by HuskySmile** [comment]
Я все эффекты с ютуба писал, делюсь своим плейлистом может кому что и понравится
https://www.youtube.com/playlist?list=PLCMt0wzDgj_io5gko_5_plL1CKM0zFl6m
